### DISCUSSION TITLE
in single req cant fetch data more than 500

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#3578] [Q&A]
i need to fetch more than 1000 data in a single req through api is there any option to get all the data in sngle req and any better option to fetch?

`https://server.com/api/collections/catview/records?page=1&perPage=1000`
not work

**Comment by ganigeorgiev** [comment]
The max `perPage` is not customizable at the moment (_it is in my todo to consider adding it as collection option, but remains a low priority as it needs further tests to ensure that users cannot abuse it_).
